src/nodes.ts

```diff
@@ -22,31 +23,33 @@ export type PropertyKey = Identifier | Literal;
 export type PropertyValue = AssignmentPattern | BindingIdentifier | BindingPattern | FunctionExpression;
 export type StatementListItem = Declaration | Statement;
 
+type Nullable<T> = T | null;
```

Comment:
I don't think there's anything wrong with just `| null` all over the place. It's immediately obvious exactly what the type is.